ah_bootstrap.py

```diff
@@ -157,7 +157,7 @@
 ]
 
 
-class _Bootstrapper(object):
```

Comment:
Please do not change external files, like `ah_bootstrap.py`. The file will become python3 once we switch to astropy_helpers v3.2.x

Reply:
Thanks, I've reverted `ah_bootstrap.py`.